src/test/java/qna/domain/AnswerRepositoryTest.java

```diff
@@ -0,0 +1,98 @@
+package qna.domain;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+@DataJpaTest
+public class AnswerRepositoryTest {
+
+	@Autowired
+	AnswerRepository answers;
+
+	@Test
+	@DisplayName("답변을 생성하면, ID가 존재해야 한다")
+	public void saveTest(){
```

Comment:
```suggestion
	public void saveTest() {
```

 IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.